test/CMakeLists.txt

```diff
@@ -46,3 +46,44 @@ foreach(GTEST_SOURCE_file ${tests})
 
    set_tests_properties(${BINARY_NAME} PROPERTIES TIMEOUT 240)
 endforeach()
+
+find_package(PythonInterp 3 REQUIRED)
+
+if(NOT PYTHONINTERP_FOUND)
+  message(STATUS "No PythonInterp found. Linters will not be executed")
+  return()
+endif()
+
+find_program(WGET_EXE wget)
+if(WGET_EXE)
+  message(STATUS "Found WGet: ${WGET_EXE}")
+  add_custom_target(wget_cppcheck
+                    COMMAND ${WGET_EXE} -q -O cppcheck.py https://raw.githubusercontent.com/ament/ament_lint/master/ament_cppcheck/ament_cppcheck/main.py
+  )
+
+  add_test(NAME console_bridge_cppcheck
+           COMMAND "${PYTHON_EXECUTABLE}"
+                   "cppcheck.py"
+                   "--language=c++" "${CMAKE_SOURCE_DIR}/src" "${CMAKE_SOURCE_DIR}/include"
+  )
+
+  add_custom_target(wget_cpplint
+                    COMMAND wget -q -O cpplint.py https://raw.githubusercontent.com/ament/ament_lint/master/ament_cpplint/ament_cpplint/cpplint.py
+  )
+
+  add_test(NAME console_bridge_cpplint
+           COMMAND "${PYTHON_EXECUTABLE}"
+                   "cpplint.py"
+                   "--counting=detailed"
+                   "--extensions=cpp,h"
+                   "--linelength=100"
+                   "--filter=-build/c++11,-runtime/references,-whitespace/braces,-whitespace/indent,-whitespace/parens,-whitespace/semicolon"
+           "${CMAKE_SOURCE_DIR}/include/console_bridge/console.h"
```

Comment:
Nitpick: if you use `CMAKE_SOURCE_DIR`, the tests will fail if the project is included in a bigger project via `add_subdirectory`, for example using FetchContent. Using `PROJECT_SOURCE_DIR` in this context is probably more robust.